torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -111,33 +111,54 @@ void syncStreams(
 
 } // namespace
 
+const int64_t ProcessGroupNCCL::kWatchdogThreadSleepMillis = 100;
```

Comment:
@mrshenli Line 114 and Line 116 are the only changes. I've changed from `constexpr` to `const`

Reply:
Why not constexpr in the header?

Reply:
Right -- but then they can be defined in the header as well right? It's only a numerical constant so it doesn't have any significant impact, but this way it can't be inlined.